fluent_bit/assets/configuration/spec.yaml

```
@@ -0,0 +1,21 @@
+name: Fluent Bit
+files:
+- name: fluent_bit.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - name: metrics_endpoint
+      description: The URL to Fluent Bit internal metrics per loaded plugin in Prometheus format.
+      required: true
+      enabled: true
+      value:
+        type: string
+        example: http://127.0.0.1:2020/api/v1/metrics/prometheus
+        display_default: null
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.hidden: true
```

why don't you update the description of this field instead of creating a new one?
```suggestion
        openmetrics_endpoint.description: The URL to Fluent Bit internal metrics per loaded plugin in Prometheus format.
```

---

Do you mean that the `name` should be kept as `openmetrics_endpoint`, just change its description? 
I thought about it. However, new endpoints might be added in the future, and I'm not sure how to name those. For example, I also hope to include Fluent Bit's `storage` metrics in this integration in the future. They are currently available only in [JSON format](https://docs.fluentbit.io/manual/administration/monitoring#rest-api-interface).

---

Ohh, it makes sense. Thanks for the clarification